### ISSUE TITLE
System refresh should clear all filters

### ISSUE THREAD

**Issue by Chinedar** [hzi-braunschweig/SORMAS-Project#983]
The filters remain as set even after the system is refreshed.

**Comment by MartinWahnschaffeSymeda** [comment]
@Chinedar You are probably refering to filters in the case directory and other directory views in the web application, right? There is a reset button that allows you to clear all filters.

**Comment by Chinedar** [comment]
@MartinWahnschaffeSymeda Yes Martin thanks. Yes this works well but I feel that a refresh or a log-out should also reset the filters to default. If you feel otherwise then that's fine. Thanks again.
